### PR TITLE
Math Magicians: tests

### PR DESCRIPTION
**Pull request by Peter-Dumbari**
In this PR:

- We created a new branch called Test
- We react-renderer-test  on my calculator component
- We ran Jest unit test for calculate.js file
- we also ran Jest unit test for operate.js file
- then finally ran linter validation check 
- we created a Pull Request

### CODE REVIEW

File: Test/Calculator.test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import renderer from 'react-test-renderer';
+import Calculator from '../src/Components/Calculator';
+
+describe('Calculator component', () => {
+  it('renders correctly', () => {
+    const component = renderer.create(<Calculator />);
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```
Comment (Gambit142) [inline Test/Calculator.test.js:11]: - [ ] Great work using react-test-renderer to test the `Calculator` component. Based on the requirements, you ought to test all components. Kindly use it to test all major components (`Home` and `Quotes` components) created in this application. 👍 😎

Comment (Gambit142) [inline Test/Calculator.test.js:11]: - [ ] One of the requirements is to use React Testing Library to simulate user interaction in the application but it has not been executed. Kindly implement this requirement. You can learn more about React Testing Library by reading these resources:
- [Documentation](https://testing-library.com/docs/react-testing-library/intro/)
- [External Resource](https://www.robinwieruch.de/react-testing-library/) 💻 🍪

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @Peter-Dumbari @coker2019 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- Use of jest to test logic files ✔️ 
- Use of Jest's react-renderer-test to test components ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: Test/Calculator.test.js (hunk 1 above)
Comment (peterrobert) [reply Test/Calculator.test.js:11]: This change has been done well

Comment (peterrobert) [reply Test/Calculator.test.js:11]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @Peter-Dumbari, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**